### PR TITLE
Better score screen

### PR DESCRIPTION
**Pull request by Boonapart228**

### CODE REVIEW

File: app/src/main/java/com/balan/androidquestionsapp/domain/models/SortDirections.kt [hunk 1]
```diff
@@ -8,7 +8,10 @@ import androidx.compose.ui.graphics.vector.ImageVector
...
 enum class SortDirections(val textId: Int, val icon: ImageVector) {
-    INCREASING(textId = R.string.increasing, icon = Icons.Default.KeyboardArrowDown),
-    DECREASING(textId = R.string.decreasing, icon = Icons.Default.KeyboardArrowUp ),
-    NAME(textId = R.string.name,icon = Icons.Default.Person)
+    INCREASING(
+        textId = R.string.increasing,
+        icon = Icons.Default.KeyboardArrowDown,
+    ),
+    DECREASING(textId = R.string.decreasing, icon = Icons.Default.KeyboardArrowUp),
+    NAME(textId = R.string.name, icon = Icons.Default.Person),
```
Comment (NAVIRKAAAA) [inline app/src/main/java/com/balan/androidquestionsapp/domain/models/SortDirections.kt:16]: removed from this enum class

File: app/src/main/java/com/balan/androidquestionsapp/presentation/score/components/ScoreContent.kt [hunk 2]
```diff
@@ -50,6 +52,7 @@ fun ScoreContent(
     onSortClick: (SortDirections) -> Unit,
     onActiveClick: () -> Unit,
     onConfirmationClick: (DialogAction) -> Unit,
+    onActive: (SortDirections) -> Unit,
```
Comment (NAVIRKAAAA) [inline app/src/main/java/com/balan/androidquestionsapp/presentation/score/components/ScoreContent.kt:55]: change name

File: app/src/main/java/com/balan/androidquestionsapp/presentation/score/components/ScoreContent.kt [hunk 3]
```diff
@@ -72,7 +76,8 @@ fun ScoreContent(
                     user = user,
                     onDeleteUserScoreClick = { viewModel.onDeleteScoreClick(user) },
                     getScore = { viewModel.getScore(user) },
-                    isTestPassed = viewModel::isTestPassed
+                    isTestPassed = viewModel::isTestPassed,
+                    isLoading = state.isLoader
```
Comment (NAVIRKAAAA) [inline app/src/main/java/com/balan/androidquestionsapp/presentation/score/components/ScoreContent.kt:80]: rename

File: app/src/main/java/com/balan/androidquestionsapp/presentation/score/components/ScoreContent.kt [hunk 4]
```diff
@@ -85,68 +90,136 @@ fun ScoreContent(
...
                     Icon(
                         imageVector = Icons.Filled.Delete,
                         contentDescription = null,
                     )
                 }
-
             }
         }
     }
+
```
Comment (Boonapart228) [inline app/src/main/java/com/balan/androidquestionsapp/presentation/score/components/ScoreContent.kt:222]: ppppppp

File: app/src/main/java/com/balan/androidquestionsapp/presentation/score/components/ScoreContent.kt [hunk 5]
```diff
@@ -85,68 +90,136 @@ fun ScoreContent(
...
 }
```
Comment (Boonapart228) [inline app/src/main/java/com/balan/androidquestionsapp/presentation/score/components/ScoreContent.kt:223]: ppp

File: app/src/main/java/com/balan/androidquestionsapp/presentation/score/components/ScoreContent.kt [hunk 6]
```diff
@@ -85,68 +90,136 @@ fun ScoreContent(
...
                 ) {
                     Icon(
                         imageVector = Icons.Filled.Delete,
                         contentDescription = null,
                     )
                 }
-
             }
         }
     }
```
Comment (Boonapart228) [inline app/src/main/java/com/balan/androidquestionsapp/presentation/score/components/ScoreContent.kt:221]: ppp

File: app/src/main/java/com/balan/androidquestionsapp/presentation/score/components/ScoreContent.kt [hunk 7]
```diff
@@ -205,7 +277,8 @@ fun ScoreContentPreview() {
             )
         ),
         onDeleteUserScoreClick = {},
-        getScore = { 5 },
-        isTestPassed = { false },
+        getScore = { 10 },
+        isTestPassed = { true },
+        isLoading = false
     )
```
Comment (Boonapart228) [inline app/src/main/java/com/balan/androidquestionsapp/presentation/score/components/ScoreContent.kt:283]: ppp

File: app/src/main/java/com/balan/androidquestionsapp/domain/models/SortDirections.kt (hunk 1 above)
Comment (Boonapart228) [reply app/src/main/java/com/balan/androidquestionsapp/domain/models/SortDirections.kt:16]: Поки залишу так

File: app/src/main/java/com/balan/androidquestionsapp/domain/models/SortOption.kt [hunk 8]
```diff
@@ -7,8 +7,11 @@ import androidx.compose.material.icons.filled.Person
...
-enum class SortDirections(val textId: Int, val icon: ImageVector) {
-    INCREASING(textId = R.string.increasing, icon = Icons.Default.KeyboardArrowDown),
-    DECREASING(textId = R.string.decreasing, icon = Icons.Default.KeyboardArrowUp ),
-    NAME(textId = R.string.name,icon = Icons.Default.Person)
+enum class SortOption(val textId: Int, val icon: ImageVector) {
+    INCREASING(
+        textId = R.string.increasing,
+        icon = Icons.Default.KeyboardArrowDown,
+    ),
+    DECREASING(textId = R.string.decreasing, icon = Icons.Default.KeyboardArrowUp),
```
Comment (Boonapart228) [inline app/src/main/java/com/balan/androidquestionsapp/domain/models/SortOption.kt:15]: wqew

**Pull request merged**